packages/cashier/src/pages/payment-agent-transfer/payment-agent-transfer-form/__tests__/payment-agent-transfer-form.spec.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable testing-library/no-container,testing-library/no-node-access */
```

Comment:
is this line needed, can we remove it, and we can look for some alternative for container methods, getByRole or pass data-testid and then use getByTestId

Reply:
I tested the `data-testid` and didn't work. we need to find an alternative for this. But because it will need to change in components as well, I think it would be better to apply those changes in a separate PR for better testing and checking.
What do you think @nijil-binary?